test/server/data/test-suites/regression-gh-599/testfile.js

```diff
@@ -0,0 +1,28 @@
+import { APIError } from '../../../../../lib/errors/runtime';
+
+fixture `Test`
+    .page `http://example.com`;
+
+var obj = {
+    method1: () => {
+        var err = new APIError('method1');
+
+        // NOTE: to preserve callsite
+        err.isTestCafeError = true;
```

Comment:
Simple error will have her relational callsite, in other words it would be:
```js
" > 8 |        throw new Error('yo') \n"
```
We need to track `method1`, because its callsite was wrong originally.